src/v/cluster/self_test_backend.cc

```diff
@@ -79,19 +94,18 @@ get_status_response self_test_backend::start_test(start_test_request req) {
     if (units) {
         _id = req.id;
         vlog(
-          clusterlog.info, "Starting redpanda self-tests with id: {}", req.id);
+          clusterlog.debug, "Request to start self-tests with id: {}", req.id);
         (void)do_start_test(req.dtos, req.ntos)
```

Comment:
This line wasn't added in this PR, but rather than using `(void)` to background a coroutine that includes a gate holder, use ssx::spawn_with_gate